src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -243,40 +247,40 @@ class LicenceService(
     }
 
     recordLicenceEventForStatus(licenceId, updatedLicence, request)
-    auditStatusChange(licenceId, updatedLicence, request)
+    auditStatusChange(updatedLicence, request.username, request.fullName)
   }
 
-  private fun auditStatusChange(licenceId: Long, licenceEntity: EntityLicence, request: StatusUpdateRequest) {
-    val fullName = "${licenceEntity.forename} ${licenceEntity.surname}"
+  private fun auditStatusChange(licenceEntity: EntityLicence, username: String, fullName: String?) {
+    val licenceFullName = "${licenceEntity.forename} ${licenceEntity.surname}"
     val detailText =
-      "ID $licenceId type ${licenceEntity.typeCode} status ${licenceEntity.statusCode.name} version ${licenceEntity.version}"
+      "ID ${licenceEntity.id} type ${licenceEntity.typeCode} status ${licenceEntity.statusCode.name} version ${licenceEntity.version}"
 
     val summaryText = when (licenceEntity.statusCode) {
-      APPROVED -> "Licence approved for $fullName"
-      REJECTED -> "Licence rejected for $fullName"
-      IN_PROGRESS -> "Licence edited for $fullName"
-      ACTIVE -> "Licence set to ACTIVE for $fullName"
-      INACTIVE -> "Licence set to INACTIVE for $fullName"
-      VARIATION_IN_PROGRESS -> "Licence variation changed to in progress for $fullName"
-      VARIATION_APPROVED -> "Licence variation approved for $fullName"
-      VARIATION_REJECTED -> "Licence variation referred for $fullName"
+      APPROVED -> "Licence approved for $licenceFullName"
+      REJECTED -> "Licence rejected for $licenceFullName"
+      IN_PROGRESS -> "Licence edited for $licenceFullName"
+      ACTIVE -> "Licence set to ACTIVE for $licenceFullName"
+      INACTIVE -> "Licence set to INACTIVE for $licenceFullName"
+      VARIATION_IN_PROGRESS -> "Licence variation changed to in progress for $licenceFullName"
+      VARIATION_APPROVED -> "Licence variation approved for $licenceFullName"
+      VARIATION_REJECTED -> "Licence variation referred for $licenceFullName"
       else -> "Check - licence status not accounted for in auditing"
     }
 
     auditEventRepository.saveAndFlush(
       AuditEvent(
-        licenceId = licenceId,
-        username = request.username,
-        fullName = request.fullName,
-        eventType = getAuditEventType(request),
+        licenceId = licenceEntity.id,
```

Comment:
If we only need the ID for the updated licence, could the passed param just be `updatedLicence.id`?